tests/constraint/test_TSR.cpp

```diff
@@ -18,15 +18,15 @@ using dart::common::make_unique;
 using DefaultRNG = RNGWrapper<std::default_random_engine>;
 
 
-static Eigen::Vector6d getPose(Eigen::Isometry3d isometry)
+static inline Eigen::Vector6d getPose(Eigen::Isometry3d isometry)
```

Comment:
Does this really cause warnings? This function is defined in source code so we shouldn't worry about duplicated definitions. Am I missing something here?